src/common/hist_util.h

```diff
@@ -506,7 +515,7 @@ class ParallelGHistBuilder {
   // Add new elements if needed, mark all hists as unused
   // targeted_hists - already allocated hists which should contain final results after Reduce() call
   void Reset(size_t nthreads, size_t nodes, const BlockedSpace2d& space,
-             const std::vector<GHistRow>& targeted_hists) {
+             const std::vector<GHistRowT >& targeted_hists) {
```

Comment:
An extra space here and in many other places for ```GHistRowT``` usage.

Reply:
deleted. And the same for `GradientPairT`